scripts/machine-launch-script.sh

```diff
@@ -0,0 +1,305 @@
+#!/bin/bash
+
```

Comment:
Why don't we just force the user to install Conda before running this script, instead of installing it for them?